UnitsNet/CustomCode/Wrappers/ReferencePressure.cs

```diff
@@ -36,7 +36,7 @@ public struct ReferencePressure
         ///     Gets a list of <see cref="PressureReference" /> options: <see cref="PressureReference.Gauge" />,
         ///     <see cref="PressureReference.Absolute" />, and <see cref="PressureReference.Vacuum" />
         /// </summary>
-        public static IReadOnlyList<PressureReference> References { get; } =
+        public static List<PressureReference> References { get; } =
```

Comment:
Sorry I forgot about net40. May I suggest using `PressureReference[]` instead of List? At least that prevents anyone from adding/removing/clearing the list, but the contents of the array can still be manipulated.

Reply:
That's a fair consideration. Since there should be no changes to this anyway, a `List` would not really offer much benefit that I can think of.